x-pack/plugins/apm/server/index.ts

```diff
@@ -54,6 +54,7 @@ const configSchema = schema.object({
     onboarding: schema.string({ defaultValue: 'apm-*' }),
   }),
   forceSyntheticSource: schema.boolean({ defaultValue: false }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
Does this mean we have to deal with plugins possibly being disabled again?

Reply:
Not in the "traditional" sense, where we need to test combinations of enabled/disabled... this is _specifically_ to accomplish a functional goal, along with other disabling as a group.  I've proposed making this "proprietary" in some way.